types/react-native/index.d.ts

```diff
@@ -6916,22 +6924,61 @@ type AccessibilityEvent = AccessibilityChangeEvent | AccessibilityAnnoucementFin
  * @see https://facebook.github.io/react-native/docs/accessibilityinfo.html
  */
 export interface AccessibilityInfoStatic {
+    /**
+     * Query whether bold text is currently enabled.
+     *
+     * @platform ios
+     */
+    isBoldTextEnabled: () => Promise<boolean>;
+
+    /**
+     * Query whether grayscale is currently enabled.
+     *
+     * @platform ios
+     */
+    isGrayscaleEnabled: () => Promise<boolean>;
+
+    /**
+     * Query whether invert colors is currently enabled.
+     *
+     * @platform ios
+     */
+    isInvertColorsEnabled: () => Promise<boolean>;
+
+    /**
+     * Query whether reduce motion is currently enabled.
+     */
+    isReduceMotionEnabled: () => Promise<boolean>;
+
+    /**
+     * Query whether reduce transparency is currently enabled.
+     *
+     * @platform ios
+     */
+    isReduceTransparencyEnabled: () => Promise<boolean>;
+
     /**
      * Query whether a screen reader is currently enabled.
-     * Returns a promise which resolves to a boolean. The result is true when a screen reader is enabled and false otherwise.
      */
-    fetch: () => Promise<boolean>;
+    isScreenReaderEnabled: () => Promise<boolean>;
+
+    /**
+     * Query whether a screen reader is currently enabled.
+     * 
```

Comment:
The [Travis CI failure](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/570049616#L560) indicates that there’s trailing whitespace here.

```suggestion
     *
```

Reply:
🤦‍♂  sorry!